### ISSUE TITLE
Table format broken on Windows

### ISSUE THREAD

**Issue by daniel-liuzzi** [paulirish/git-recent#8]
At the expense of being the odd one out around here on Windows, I installed git-recent on my system and already love it. It works perfectly overall, except for one minor thing. `column` messes up the table layout when there are control chars in any except the last column. For example:

![git-recent](https://cloud.githubusercontent.com/assets/143884/17586600/014d0cb8-5fc3-11e6-884e-a5ba503df522.png)

If I go and temporarily remove the coloring from the refname like so:

``` diff
  format="\
- %(HEAD) %(color:yellow)%(refname:short)%(color:reset)|\
+ %(HEAD) %(refname:short)|\
  %(color:bold red)%(objectname:short) %(color:bold green)(%(committerdate:relative)) %(color:blue)%(authorname) %(color:reset)%(color:yellow)%(upstream:track)%0a\
-   %(color:black) %(color:reset)|%(contents:subject)%0a\
+    |%(contents:subject)%0a\
    |"
```

Then columns _do_ line up properly and match the screenshot on the README:

![git-recent-2](https://cloud.githubusercontent.com/assets/143884/17586725/b48f25d6-5fc3-11e6-8715-56d36bb432ea.png)

If I have to guess, this seems to be more of a `column` problem than a `git recent` problem, but I though I bring it up in case I'm missing something. If anyone can point me in the right direction I can make a PR attempt.

**Comment by paulirish** [comment]
how about moving yellow up a line and reset to right before the bold/red?
Curious if that gets us consitent rendering across all the column's

like this maybe?

``` sh
format="\%(color:yellow)
%(HEAD) %(refname:short)|\
%(color:reset)%(color:bold red)%(objectname:short) %(color:bold green)(%(committerdate:relative)) %(color:blue)%(authorname) %(color:reset)%(color:yellow)%(upstream:track)%0a\
  %(color:black) %(color:reset)|%(contents:subject)%0a\
  |"
```

**Comment by daniel-liuzzi** [comment]
Unfortunately it didn't work :confused: It also lost the yellow coloring and rendered a `\` above the refname:

![git-recent-1](https://cloud.githubusercontent.com/assets/143884/18026081/81cf85e8-6c3c-11e6-8431-e93293b6b826.png)

I've been playing with this some more and got an idea. Give me a few minutes and I'll sent a PR.

**Comment by daniel-liuzzi** [comment]
Fixed in #9.
